xyz-jobs/xyz-job-steps/src/main/java/com/here/xyz/jobs/steps/impl/S3MetricsCollectorStep.java

```diff
@@ -0,0 +1,145 @@
+package com.here.xyz.jobs.steps.impl;
+
+import com.fasterxml.jackson.annotation.JsonView;
+import com.here.xyz.XyzSerializable;
+import com.here.xyz.jobs.steps.Step;
+import com.here.xyz.jobs.steps.execution.SyncLambdaStep;
+import com.here.xyz.jobs.steps.inputs.Input;
+import com.here.xyz.jobs.steps.inputs.InputFromOutput;
+import com.here.xyz.jobs.steps.inputs.UploadUrl;
+import com.here.xyz.jobs.steps.outputs.FeatureStatistics;
+import com.here.xyz.jobs.steps.outputs.Output;
+import com.here.xyz.models.hub.Ref;
+import com.here.xyz.util.service.BaseHttpServerVerticle;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class S3MetricsCollectorStep extends SyncLambdaStep {
+    public static final String S3_METRICS = "s3_metrics";
```

Comment:
The name of that outputSet is user-facing. It should not be related to "S3". It could be something like "statistics" or so. Then the user could acess it through the API under the endpoint `/jobs/{jobId}/outputs/statistics`.

Reply:
done